# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/test/mochitest.ini
# Commit: 10d327bb5893
# Full Hash: 10d327bb5893211f030222b02456bfc11e81a943
# Author: Olli Pettay <Olli.Pettay@helsinki.fi>
# Date: 2021-05-18 21:36:28
# Regressor Bug: 1705080
# File Overlap Count: 1
# Description:
#   Bug 1705080, make MediaStreams behave as normal EventTargets, r=jib
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D115269
# ==============================================================================

diff -r 802480d6b25b -r 10d327bb5893 dom/media/test/mochitest.ini
--- a/dom/media/test/mochitest.ini	Tue May 18 16:19:44 2021 +0000
+++ b/dom/media/test/mochitest.ini	Tue May 18 16:21:27 2021 +0000
@@ -963,6 +963,7 @@
 tags=mtg
 [test_mediarecorder_record_getdata_afterstart.html]
 tags=mtg capturestream
+[test_mediastream_as_eventarget.html]
 [test_mediatrack_consuming_mediaresource.html]
 [test_mediatrack_consuming_mediastream.html]
 scheme=https